nomos-cli/src/api/mempool.rs

```diff
@@ -2,14 +2,14 @@ use super::CLIENT;
 use reqwest::{Error, Response, Url};
 use serde::Serialize;
 
-pub async fn send_certificate<C>(node: &Url, cert: &C) -> Result<Response, Error>
+pub async fn send_blob_info<I>(node: &Url, info: &I) -> Result<Response, Error>
 where
-    C: Serialize,
+    I: Serialize,
 {
     const NODE_CERT_PATH: &str = "mempool/add/cert";
     CLIENT
         .post(node.join(NODE_CERT_PATH).unwrap())
-        .json(cert)
+        .json(info)
         .send()
         .await
 }
```

Comment:
We still have this. But we should remove it after we have the executor swarm. In where we should plug the mempool gossip network :thinking:

Reply:
I'm not sure I get it. At the moment the mempool gossip network is between nodes as I understand. Once the item is added (in this case via http), the node will gossip it. I don't think we have a way to add items via libp2p currently.

Reply:
Yeah, you are partially right. In this case we are emulating an executor, an executor should be participating as a validator itself so we could just go ahead and participate in the network here. For now this is fine though.